### ISSUE TITLE
Simplify

### ISSUE THREAD

**Issue by quebeh** [madrasah-techonatura/Useless-repository-made-by-soy#2]
Please simplify the repository and sort it to categories for example Assets, The Games, Prefabs etc.

**Comment by Aldhanekaa** [comment]
Yes. You probably have to learn git if you don't want to struggle to upload code
